junit-runtime/src/main/scala/org/scalajs/junit/JUnitMasterRunner.scala

```diff
@@ -0,0 +1,82 @@
+package org.scalajs.junit
+
+import com.novocode.junit.RunSettings
+import sbt.testing._
+import java.util.concurrent.atomic.AtomicInteger
+
+final class JUnitMasterRunner(
+    args: Array[String],
+    remoteArgs: Array[String],
+    testClassLoader: ClassLoader,
+    runSettings: RunSettings)
+    extends JUnitBaseRunner(args, remoteArgs, testClassLoader, runSettings) {
+
+  private[this] val registeredCount = new AtomicInteger(0)
+
+  private[this] val doneCount = new AtomicInteger(0)
+  private[this] val passedCount = new AtomicInteger(0)
+  private[this] val failedCount = new AtomicInteger(0)
+  private[this] val ignoredCount = new AtomicInteger(0)
+  private[this] val skippedCount = new AtomicInteger(0)
+  private[this] val totalCount = new AtomicInteger(0)
+
+  private[this] val slaveCount = new AtomicInteger(0)
```

Comment:
Similarly, managing `slaveCount` is useless in a real testing framework, I believe.